React/Base/RCTBundleURLProvider.m

```diff
@@ -19,9 +19,19 @@
 static NSString *const kRCTEnableMinificationKey = @"RCT_enableMinification";
 
 static NSString *const kDefaultPort = @"8081";
+static NSString *ipGuess;
 
 @implementation RCTBundleURLProvider
 
++ (void)initialize
+{
+#if RCT_DEV
```

Comment:
You should put this outside the `+ (void)initialize`, otherwise we'll still be executing an empty method in prod.